pkgs/development/compilers/ocaml/ber-metaocaml.nix

```diff
@@ -34,9 +34,7 @@ stdenv.mkDerivation rec {
 
   prefixKey = "-prefix ";
   configureFlags = optionals useX11
-    [ "-x11lib" x11lib
-      "-x11include" x11inc
-      "-flambda"
+    [ "--enable-flambda"
```

Comment:
Why is this `flambda` option guarded by the `useX11` condition? It seems unrelated…

Reply:
The formatting is now also weird. This could easily be in one line.

Reply:
> The formatting is now also weird. This could easily be in one line.

Fixed.

Reply:
> Why is this `flambda` option guarded by the `useX11` condition? It seems unrelated…

I'm afraid I have no idea.  I was hesitant to make a functionality change in a version-bumping PR, but if you prefer I would be happy to remove the `--enable-flambda`.